### PR TITLE
add tab index

### PR DESCRIPTION
**Pull request by z-vr**
- dist is required if package is installed from github
- add tab-index property to be able to control from keyboard
- add yarn.lock

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,3 @@
 node_modules
 .DS_Store
 npm-debug.log
-dist/*.js
```
Comment (karltaylor) [inline .gitignore]: best to keep build files out of the repo

File: dist/Collapsible.js [hunk 2]
```diff
@@ -0,0 +1,293 @@
+'use strict';
```
Comment (karltaylor) [inline dist/Collapsible.js]: Same again, best to keep `dist/*` in .gitignore

**Comment by karltaylor** [comment]
This look alright if we just add the `dist/*.js` file to the `.gitignore` again.

I also looked into how this could be extended - https://developers.google.com/web/fundamentals/accessibility/focus/using-tabindex#managing_focus_in_components

Being able to `focus` to the next Collapsible element and open it would be cool but that's probably quite a bit of tinkering.

Another thing would be to enable this as an option, or having the default prop as `null`. Since adding `tabIndex=0` makes the element focusable and as the blue border in Chrome by default. People would have to then add `outline: none` to the css to remove it.

**Comment by jaredlevi** [comment]
Is there any way we can get this merged in. I really need the tab index to make my project accessible for keyboard users? Thanks

**Comment by karltaylor** [comment]
@jaredlevi feel free to fix this PR. I don't think `tabIndex` should be enabled by default. If you don't get round to it I can have a look over the weekend

**Comment by karltaylor** [comment]
Closed in favour of #92

**Comment by z-vr** [comment]
@karltaylor hey sorry didn't get back to this PR, was real busy. Glad a similar PR been merged.
also just FYI, by not keeping a build in this repo, it's impossible to install it as github dependency rather than NPM one, which I guess is really only useful when forking anyway 👋

**Comment by karltaylor** [comment]
@z-vr No worries! - If you want to install via github and use it in your `package.json` you can always do a postinstall hook to install the devDeps and then build the package? Could that not work for you?

**Pull request closed without merging**